src/SFA.DAS.EAS.Employer_Financial.Database/Scripts/PreDeployment/AML-2505-PopulateNullProviderNames.sql

```diff
@@ -0,0 +1,49 @@
+﻿BEGIN TRY
```

Comment:
I can't see an `ORDER BY` clause, I think you'll need one to pull out the latest provider name?

Reply:
I've added the logic to get the latest not null value in and tested it on my local with data that was manual setup up to recreated the issue. Can you check to make sure you're happy with it then hopefully this can get added to the branch.

Reply:
Spoke with Elliot and he's now fine with the branch so merging into AML-2365